src/components/navbar.jsx

```diff
@@ -10,8 +10,8 @@ const Navbar = () => {
                  </div>
 
                 <div className='flex items-center md:flex'>
-                    <a href="https://github.com/Njong392/Abbreve"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" className='w-12 h-12 hover:rotate-6 hover:scale-110 mr-4'><path fill="none" d="M0 0h24v24H0z"/><path d="M12 2C6.475 2 2 6.475 2 12a9.994 9.994 0 0 0 6.838 9.488c.5.087.687-.213.687-.476 0-.237-.013-1.024-.013-1.862-2.512.463-3.162-.612-3.362-1.175-.113-.288-.6-1.175-1.025-1.413-.35-.187-.85-.65-.013-.662.788-.013 1.35.725 1.538 1.025.9 1.512 2.338 1.087 2.912.825.088-.65.35-1.087.638-1.337-2.225-.25-4.55-1.113-4.55-4.938 0-1.088.387-1.987 1.025-2.688-.1-.25-.45-1.275.1-2.65 0 0 .837-.262 2.75 1.026a9.28 9.28 0 0 1 2.5-.338c.85 0 1.7.112 2.5.337 1.912-1.3 2.75-1.024 2.75-1.024.55 1.375.2 2.4.1 2.65.637.7 1.025 1.587 1.025 2.687 0 3.838-2.337 4.688-4.562 4.938.362.312.675.912.675 1.85 0 1.337-.013 2.412-.013 2.75 0 .262.188.574.688.474A10.016 10.016 0 0 0 22 12c0-5.525-4.475-10-10-10z" fill="rgba(172,88,245,1)"/></svg></a>
-                    <Link to="form" smooth={true} duration={600} className='bg-purple text-ash font-bold rounded-xl p-2 hover:scale-110 outline-none hidden md:block ml-4 cursor-pointer'>Get Started </Link>
+                    <a href="https://github.com/Njong392/Abbreve"><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" className='w-12 h-12 hover:rotate-6 hover:scale-105 fill-purple hover:fill-purpleHover mr-4 duration-300 ease-in-out'><path fill="none" d="M0 0h24v24H0z"/><path d="M12 2C6.475 2 2 6.475 2 12a9.994 9.994 0 0 0 6.838 9.488c.5.087.687-.213.687-.476 0-.237-.013-1.024-.013-1.862-2.512.463-3.162-.612-3.362-1.175-.113-.288-.6-1.175-1.025-1.413-.35-.187-.85-.65-.013-.662.788-.013 1.35.725 1.538 1.025.9 1.512 2.338 1.087 2.912.825.088-.65.35-1.087.638-1.337-2.225-.25-4.55-1.113-4.55-4.938 0-1.088.387-1.987 1.025-2.688-.1-.25-.45-1.275.1-2.65 0 0 .837-.262 2.75 1.026a9.28 9.28 0 0 1 2.5-.338c.85 0 1.7.112 2.5.337 1.912-1.3 2.75-1.024 2.75-1.024.55 1.375.2 2.4.1 2.65.637.7 1.025 1.587 1.025 2.687 0 3.838-2.337 4.688-4.562 4.938.362.312.675.912.675 1.85 0 1.337-.013 2.412-.013 2.75 0 .262.188.574.688.474A10.016 10.016 0 0 0 22 12c0-5.525-4.475-10-10-10z"/></svg></a>
+                    <Link to="form" smooth={true} duration={600} className='bg-purple text-ash font-bold rounded-xl py-2 px-5 hover:bg-purpleHover hover:scale-105 outline-none hidden md:block ml-4 cursor-pointer duration-300 ease-in-out'>Get Started </Link>
```

Comment:
I think the `Get Started` button should not have the hover effect yet. When the user hovers, the contrast between the active link and the new hover effect you added does not look very good. While we are still trying to fix the contrast issues, I'll suggest you not add new background color on hover for now.